lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -17,11 +17,31 @@ export default function LMSGrader({
   children,
   assignmentName,
   courseName,
-  students,
+  students: unorderedStudents,
 }) {
   // No initial current student selected
   const [currentStudentIndex, setCurrentStudentIndex] = useState(-1);
 
+  // A sorted list of students. Students are sorted by
+  // displayName.
+  const [students] = useState(() => {
+    // Make a copy
+    const students_ = [...unorderedStudents];
+    students_.sort((student1, student2) => {
+      function compareNames(name1, name2) {
```

Comment:
I think it wise to define this function outside of the `Array.prototype.sort` call.